src/routes/users/fetch-users.e2e.test.js

```diff
@@ -0,0 +1,57 @@
+const { expect } = require('chai')
+const { fixtures, setupDB } = require('../../../test/utils')
+
+describe('GET /users', () => {   
+	describe('Failure', ()=> {
+		let userUser
+		let accessToken
+		before(async () =>{
+			userUser = await fixtures.insertUser({
+				role : 'user'
+			}) 
+			const body = { id: userUser.id, email: userUser.email }
+			accessToken = fixtures.generateAccessToken(
+				{user : body}
+			)
+		})
+		it('should return 403 if user is not an admin', async() =>{
+			const expectedError = {
+				"status":"failed",
+				"message":"Only admin can create users"
+			}
+			fixtures.api()
```

Comment:
needs a return else it will pass no matter the status code
```suggestion
			return fixtures.api()
```

Reply:
wonderful